features/datachoices/pom.xml

```diff
@@ -97,6 +97,21 @@
             <artifactId>org.opennms.features.bsm.persistence.api</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.opennms.core.ipc.sink</groupId>
+            <artifactId>org.opennms.core.ipc.sink.common</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.opennms.core.ipc.rpc</groupId>
+            <artifactId>org.opennms.core.ipc.rpc.common</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
```

Comment:
Do we need to add these dependencies in features as well ?

Reply:
I haven't had any issues building it as-is

Reply:
Better to include these bundles in datachoices features.